app/uk/gov/hmrc/channelpreferences/model/preferences/EmailAddress.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.channelpreferences.model.preferences
+
+import play.api.libs.json.{ Format, Json }
+
+case class EmailAddress(value: String) extends AnyVal
+
+object EmailAddress {
+  implicit val format: Format[EmailAddress] = Json.valueFormat[EmailAddress]
```

Comment:
could we benefit from using our EmailAddress library? the preference service (well all of our services) use it. https://github.com/hmrc/emailaddress/blob/main/src/main/scala/uk/gov/hmrc/emailaddress/EmailAddress.scala#L19

Reply:
and the formats are here: https://github.com/hmrc/emailaddress/blob/main/src/main/scala/uk/gov/hmrc/emailaddress/PlayJsonFormats.scala#L22

Reply:
good point, maybe we can leave this to the implementer of the verification user story?

Reply:
yes good point 😄